tests/relay_integration/test_integration.py

```diff
@@ -216,3 +216,23 @@ def test_transaction(self):
                     "total.time": {"unit": "millisecond", "value": pytest.approx(1050, rel=1e3)},
                 }
             }
+
+    def test_project_abuse_quotas(self):
+        # Windows < 10 seconds might be kinda flaky,
+        # so this is just a pretty basic test that sets a negative limit
+        # to reject all errors.
+        self.organization.update_option("project-abuse-quota.error-limit", -1)
+
+        event_data = {
+            "type": "error",
+            "message": "hi",
+            "timestamp": iso_format(before_now(seconds=1)),
+        }
+        event = self.post_and_retrieve_event(event_data)
+
+        # this shouldn't be passing.
+        # does this reject-all quota work?
```

Comment:
@jan-auer I know you suggested 0, but I want to try out -1 reject-all and it isn't working. Do you know if given that quota, this behavior is supported?